StudentManagement/StudentManagement.Console/Program.cs

```diff
@@ -61,8 +90,23 @@
 
             }
 
+
+            break;
+        case 3:
```

Comment:
What is the reason for this case? It doesn't make sense to let user create an empty file. It would be better to create it when you need to store the data. Please remove it

Reply:
Now in case 3 i save all students to json file and write a file